src/groups/mqb/mqba/mqba_authenticator.h

```diff
@@ -0,0 +1,168 @@
+// Copyright 2025 Bloomberg Finance L.P.
+// SPDX-License-Identifier: Apache-2.0
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// mqba_authenticator.h                                           -*-C++-*-
+#ifndef INCLUDED_MQBA_AUTHENTICATOR
+#define INCLUDED_MQBA_AUTHENTICATOR
+
+/// @file mqba_authenticator.h
+///
+/// @brief
```

Comment:
I think this documentation comment will be the most helpful of all of them.  Fine to have it blank as you're working but I think this is going to be a treasure-trove of useful info.  At least, `@brief` will be good, so the libmqb documentation has a description for this component.